GenerateAspNetCoreClient.Command/ClientModelBuilder.cs

```diff
@@ -177,12 +178,36 @@ private List<Parameter> GetParameters(ApiDescription apiDescription)
                     continue;
                 }
 
+                // IFormFile[], List<IFormFile>
+                if (paramType != null)
+                {
+                    bool isFormFileArray = paramType == typeof(IFormFile[]);
+                    bool isFormFileList =
+                        paramType.IsGenericType
+                        && paramType.GetGenericTypeDefinition() == typeof(List<>)
```

Comment:
In case we _do_ need to check for list, would it work like this?
```
paramType == typeof(List<IFormFile>) ?
```

Reply:
We don't need to check for list, per your previous comments. We only need to choose an inner type, rather than stipulate the outer type beyond `IEnumerable<IFormFile>`.